.gitignore

```diff
@@ -0,0 +1,801 @@
+
+# Created by https://www.gitignore.io/api/xamarinstudio,visualstudio,visualstudiocode,xcode,android,macos,csharp,f#,fastlane,java,jetbrains,linux,monodevelop,objective-c,swift,sublimetext,unity
+
+**/.vscode*
+
+### fastlane ###
+# fastlane - A streamlined workflow tool for Cocoa deployment
+
+# fastlane specific
+fastlane/report.xml
+
+# deliver temporary files
+fastlane/Preview.html
```

Comment:
What is this?

Reply:
Hey Dan! This is to avoid adding auto-generated code from Fastlane into GitHub